shadowsocks-csharp/Controller/StartupHotkeyReg.cs

```diff
@@ -59,8 +60,12 @@ public static void RegHotkey()
                 }
 
                 foreach (var v in _hotKeyDic)
-                {
-                    HotKeys.Regist(v.Key, v.Value);
+                {
+                    if (!HotKeys.Regist(v.Key, v.Value))
```

Comment:
Thx for your advise.
I could un-register previous keys if any registering failure . 
I don't think popup the Hotkey Register Form is necessary ,this isn't a frequently used feature.
There just simply a notification ,user could fix hotkey conflict in Hotkey Register Form.